### DISCUSSION TITLE
Please make the core game with longer progression and better english

### DISCUSSION THREAD

**Discussion by AwesomeFart** [LorisYounger/VPet#394] [Translate]
The english is bad.. few examples: "Accumulate money gains" "Stop Clean Screen" . The stuff that she says randomly when she is hungry is horribly translated - it doesnt make sense in english...

**Please let somebody make a new pass for better english.**

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

The progression is dull and its saddening how the core game could be so much better. What i mostly mean is how short is the progression line. 

You should level cap gifts and certain food so its NOT more beneficial to gain exp with gifts rather than learning / research - instead **make more steps to learn interaction - like "homeschool - lvl 10" "Elementary school - lvl 20" "high school - lvl 30" "College - lvl 40"  "Research - lvl 50" and up to lvl 100** - make her grow same as human

**level cap food and gifts so i can have motivation to unlock them**

**Dont let me just "work to level up with gifts" - thats boring.**

**Same with Jobs - please add more work  - up to lvl 100 atleast.  not just 3 jobs and then just livestream from 20-100lvl... go from shitty jobs to prestige jobs every 10 levels** 

The play interaction - let me get more animations for achievements... i want to be motivated to unlock more... pat head 100 times =  "watching anime",  spend 1000$ at Better buy = "go for a walk",  have 1000$ = "go fishing" and more....

**Food should not replace play time - thats too easy** - food should not give so much mood.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
**I understand you devs want people to mod this, but the core game should be for atleast 150hours of gameplay and 100lvls - WITHOUT MODS.**

Look at games like cookie clicker and their progression its meaningfull and engaging for 100 - 120 hours, after that - lets have fun with mods. 

**I love your vpet.** I think its generally amazing, pls dont make modders do all the job - make the first 100lvls core game good. Let us english speaking people have more fun too, its hard to love the pet when its speaking bad english...
----------------------------------------------------------------------------------------------------------------------------
Let me pay you money on steam for your work - i would pay for 21-100lvl DLC  5-10€. **_I uderstand you need more animations_**, but come on its about 20-30 animations - this should not be so hard if you let people pay money for 21-100lvl DLC.

I made this Text2307 - please atleast add this to offcial english translation https://pastebin.com/mgSiqNEy
also pls change "Accumulate money gains" to "earning money"
I have no idea what  太郎~呸主人~喝药~呸喝水啦~#Taro~Bah, master~Drink medicine~Bah, drink water~:|  this is i dont know when this is triggered so i have no idea how to translate so i translated like this:
太郎~呸主人~喝药~呸喝水啦~#Salad?- Meh. Eat one apple per day? - Meh. Eat vitamine pills? - Meh. Just drink eight glasses of water per day! :|

**Comment by AwesomeFart** [comment]
i have made a mod for english translation - feel free to use it https://steamcommunity.com/sharedfiles/filedetails/?id=3047758961

**Comment by LorisYounger** [comment]
Thank you for reply

The recent English translation content was done using machine translation and simple proofreading, as my translators are busy and I have no available translators recently.
also, translators are ordinary college students, not professionals.
If you have a better English translation, welcome and thanks to Pull Request.

The experience value added by gifts is not high, according to the design, the experience value ratio obtained by purchasing gifts is 1(money) : 6(exp)
but study efficiency is 1 : 12 (exp)
More steps of learning, such as going to school and university, will be published in the second paid DLC community in the future.

Food and gifts require corresponding levels before unlocking: Good idea, I will add it in the future

So far, this game is basically developed by me alone, Painters are also outsourced, and she has her own job. she work part-time as a painter at my group. I am also trying to find more developers, but getting involved will have a break-in period. Currently, a new programmer is developing a [mod maker](https://github.com/LorisYounger/VPet.ModMaker)
This game did not have a very large development plan at the beginning, because this was initially produced as an additional content for another game

I apologize that a lot of content won't be added quickly.

**Comment by AwesomeFart** [comment]
_"The experience value added by gifts is not high, according to the design, the experience value ratio obtained by purchasing gifts is 1(money) : 6(exp)
but study efficiency is 1 : 12 (exp)"_

Its way more efficent to work - gain money and exp via gift - compare to just learn / research and spend money to eat and drink.

The method to just work and buy gifts and food is all you need to do to lvl up and gain money. Instead -  you should be timegated by needing to let her play in between of working and eating. **This will prolong the gameplay and time to reach 100lvl.**
Thats why food should not be the only source of mood ... you should be forced to let her play. This way she doesnt need to play ever in whole 100lvls - just eat and work and get gifts.

Feel free to use the language files im making alone in the mod on steam link posted above, but i am doing this alone and for fun...its gonna be slow process..
This was my first game i have actually learned how to make mods on steam for so thank you for this adventure lol.

_"More steps of learning, such as going to school and university, will be published in the second paid DLC community in the future."_
Pls add also more steps for work and play. Thank you.
